tests/app_tests/lifecycle_test.rs

```diff
@@ -1,6 +1,8 @@
 use futures::FutureExt;
 use ratings::{
     app::AppContext,
+    features::pb::common::RatingsBand,
```

Comment:
this is an artifact from when `to_kebab_case` existed so the test `pb.rs` file was removed so we didn't manually have to keep them in sync.

This is no longer necessary and the changes responsible are squashed out, but it's so hard to excise this that unfortunately this noise is gonna be here.